Classes/Flowpack/Neos/FrontendLogin/Controller/AuthenticationController.php

```diff
@@ -50,6 +70,29 @@ protected function onAuthenticationSuccess(ActionRequest $originalRequest = NULL
 		}
 	}
 
+
+	/**
+	 * Create translated FlashMessage and add it to flashMessageContainer
+	 *
+	 * @param ActionRequest $originalRequest The request that was intercepted by the security framework, NULL if there was none
+	 * @return string
+	 */
+	protected function onAuthenticationFailure(AuthenticationRequiredException $exception = null) {
+		$title = $this->getTranslationById('authentication.failure.title');
+		$message = $this->getTranslationById('authentication.failure.message');
+		$this->flashMessageContainer->addMessage(new Error($message, ($exception === null ? 1496914553 : $exception->getCode()), array(), $title));
```

Comment:
Using the FlashMessageContainer directly is discouraged in favor of the higher-level method `addFlashMessage()` =>
```php
$this->addFlashMessage($message, $title, Message::SEVERITY_ERROR, [], $exception === null ? 1496914553 : $exception->getCode())
```